### ISSUE TITLE
DebugMode

### ISSUE THREAD

**Issue by jenshelderweirdt** [chsami/charta#2]
It would be handy to enable sort of a DebugMode that logs all events of the GoogleMaps API.

https://developers.google.com/maps/documentation/javascript/events

**Comment by chsami** [comment]
I agree with this sentiment and  I'm looking forward to this feature!
